packages/core/src/system.ts

```diff
@@ -27,6 +29,9 @@ export function createSystem<T extends ActorSystemInfo>(): ActorSystem<T> {
     set: (key, actorRef) => {
       keyedActors.set(key, actorRef);
       reverseKeyedActors.set(actorRef, key);
+    },
+    stop: () => {
+      rootActor._stop?.();
     }
```

Comment:
`system` is available everywhere, so we could still stop the whole thing from an actor deep in the "tree" - this feels off

Reply:
Any other suggestions? Nothing's stopping a dev (no pun intended) from doing _anything_ in an action, so no matter where we put stop, we still have this "problem"